_topic_maps/_topic_map.yml

```diff
@@ -3820,6 +3820,8 @@ Topics:
       File: virt-hot-plugging-virtual-disks
     - Name: Expanding VM disks
       File: virt-expanding-vm-disks
+    - Name: Configuring shared volumes for virtual machines
```

Comment:
In the topic map, I would change this to "Configuring shared volumes". You don't need "for virtual machines" because the topic is under VMs and long titles can be difficult to read. The topic map name can be shortened. It does not have to match the assembly name.

Reply:
Done